flink/interpreter/src/main/java/org/apache/zeppelin/flink/HadoopUtils.java

```diff
@@ -55,6 +65,18 @@ public static String getYarnAppTrackingUrl(ClusterClient clusterClient) throws I
     return yarnClient.getApplicationReport(yarnAppId).getTrackingUrl();
   }
 
+  public static int getFlinkRestPort(String yarnAppId) throws IOException, YarnException {
+    YarnClient yarnClient = YarnClient.createYarnClient();
+    YarnConfiguration yarnConf = new YarnConfiguration();
+    // disable timeline service as we only query yarn app here.
+    // Otherwise we may hit this kind of ERROR:
+    // java.lang.ClassNotFoundException: com.sun.jersey.api.client.config.ClientConfig
+    yarnConf.set("yarn.timeline-service.enabled", "false");
+    yarnClient.init(yarnConf);
+    yarnClient.start();
+    return yarnClient.getApplicationReport(ConverterUtils.toApplicationId(yarnAppId)).getRpcPort();
```

Comment:
It is almost the same as the function above, but it would be good to separate it with the `getApplicationReport()` function.

Reply:
Good point, fixed